scene/resources/animation.cpp

```diff
@@ -83,9 +83,11 @@ bool Animation::_set(const StringName &p_name, const Variant &p_value) {
 
 				const real_t *r = values.ptr();
 
-				tt->transforms.resize(vcount / 12);
+				int transform3d_size = (int)sizeof(Transform3D);
 
-				for (int i = 0; i < (vcount / 12); i++) {
+				tt->transforms.resize(vcount / transform3d_size);
+
+				for (int i = 0; i < (vcount / transform3d_size); i++) {
```

Comment:
Why not just make the value have the division already in it?

```cpp
const int transform3d_count = vcount / sizeof(Transform3D);
tt->transforms.resize(transform3d_count);
for (int i = 0; i < transform3d_count; i++) {
```

Reply:
Cause i don't want to make a refactor. I just fix the issue. Refactor should be different commit with commit message. (also unrelated to this issue)